src/styles/blocks/logo.scss

```diff
@@ -0,0 +1,3 @@
+.logo {
+  height: 40px;
```

Comment:
instead of hard coded height, you can use display flex and align it to centre vertically